cmd/cstor-volume-mgmt/volume/volume.go

```diff
@@ -196,6 +198,57 @@ func CreateIstgtConf(cStorVolume *apis.CStorVolume) ([]byte, error) {
 	return dataBytes, nil
 }
 
+// ResizeTargetVolume sends resize volume command to istgt and get the response
+func ResizeTargetVolume(cStorVolume *apis.CStorVolume) error {
+	index, _, err := FileOperatorVar.GetLineDetails(util.IstgtConfPath, "LUN0 Storage")
+	if err != nil {
+		return errors.Wrapf(err,
+			"failed to get storage details from %s file",
+			util.IstgtConfPath)
+	} else if index == -1 {
+		return errors.Errorf("storage details not found in file %s", util.IstgtConfPath)
+	}
+	// send resize command to istgt and read the response
+	resizeCmd := getResizeCommand(cStorVolume)
+	sockResp, err := UnixSockVar.SendCommand(resizeCmd)
+	if err != nil {
+		return errors.Wrapf(
+			err,
+			"failed to execute istgt %s command on volume %s",
+			util.IstgtResizeCmd,
+			cStorVolume.Name)
+	}
+	if v1_strings.MakeList(sockResp...).Contains("ERR") {
+		err = errors.Errorf("%v", sockResp)
+		return errors.Wrapf(
+			err,
+			"failed to execute istgt %s command on volume %s",
+			util.IstgtResizeCmd,
+			cStorVolume.Name)
+	}
+
+	updateStorageVal := fmt.Sprintf("  LUN0 Storage %s 32K", cStorVolume.Spec.Capacity.String())
+	err = FileOperatorVar.Updatefile(util.IstgtConfPath, updateStorageVal, index, 0644)
```

Comment:
lets not do get details and directly do Updatefile which finds the line and updates it

Reply:
lets not do `GetLineDetails` and directly do Updatefile which tries to finds the line and updates it

Reply:
> Should we do this way, 1. read whole file content, 2. update the content and then 3. re-write the file?

Done in this way